frameworks/Elixir/phoenix/lib/hello_web/controllers/page_controller.ex

```diff
@@ -41,11 +42,11 @@ defmodule HelloWeb.PageController do
       message: "Additional fortune added at request time."
     }
 
-    fortunes = [additional_fortune | Repo.all(Fortune)]
+    fortunes =
+      [additional_fortune | Repo.all(Fortune)]
+      |> Enum.sort_by(& &1.message)
```

Comment:
Have you benchmarked this one? I actually expect `sort` to be more efficient, because `sort_by` needs to do additional passes on the data.